python/helpers/pydev/_pydev_bundle/_pydev_calltip_util.py

```diff
@@ -0,0 +1,160 @@
+'''
+Id:          "$Id$"
+Copyright:   Copyright (c) 2015 Bank of America Merrill Lynch, All Rights Reserved
```

Comment:
Actually our CLA doesn't transfer the copyright it only grants the license, but it was you, Yuli Fitterman, who signed the CLA, not Bank of America, so we assume that you are the copyright owner in this case.